packages/ihe-gateway-sdk/src/client/ihe-gateway.ts

```diff
@@ -55,7 +55,7 @@ export class IHEGateway {
    * @param documentQueryRequest An array of document query transaction requests.
    *
    */
-  async startDocumentsQuery(documentQueryRequest: DocumentQueryRequest): Promise<void> {
+  async startDocumentsQuery(documentQueryRequest: DocumentQueryRequest[]): Promise<void> {
     await this.api.post(IHEGateway.DOCUMENT_QUERY_ENDPOINT, documentQueryRequest);
```

Comment:
I suggest we don’t send (or respond with) arrays, but instead always use a JSON object.

This allows the API to extend its functionality by adding new properties without breaking clients.